web/concrete/src/Page/Type/Type.php

```diff
@@ -1102,19 +1102,14 @@ public function getPageTypeValidatorObject()
         }
     }
 
-    public function createDraft(PageTemplate $pt, $u = false)
+    public function createDraft(PageTemplate $pt)
```

Comment:
For some reason I was thinking that extra parameters passed raised an E_NOTICE, looks like that's not true.